### ISSUE TITLE
user stats

### ISSUE THREAD

**Issue by mparadinha** [ryukinix/mal#38]
Maybe we could create a command to show the user stats like those on the website.
![mal_drassof_stats](https://cloud.githubusercontent.com/assets/8689653/22016681/05ee8518-dc9f-11e6-9ec1-f74d395384b5.png)

(this isn't my MAL btw, it's of a friend of mine how has watched a lot more then me)

Something like this: 

```
$ mal stats
Watching: 3
Completed: 32
...
Total episodes: 1500
Watch time: 24.10 days
```

We could even try to do that bar graph with color, but I'm not really sure how to do that.

**Comment by ryukinix** [comment]
HMMM, great idea.

**Comment by ryukinix** [comment]
Current result of implementation: 
![image](https://cloud.githubusercontent.com/assets/7642878/22139047/9182d8a2-dec2-11e6-8148-786c8b907350.png)

I think we can merge it. But the rewatched counter is right? I think some animes I re-watched sometimes on my list...

**Comment by ryukinix** [comment]
![image](https://cloud.githubusercontent.com/assets/7642878/22139502/23956bf4-dec5-11e6-9f8f-b7a9217742e8.png)
Yes, I have 5 re-watched animes. We need check it before merge. Also episodes numbers are different.

**Comment by mparadinha** [comment]
I don't what's up with the rewatches but the episodes might be because it's not counting the rewatched ones.

**Comment by ryukinix** [comment]
This makes sense

**Comment by mparadinha** [comment]
Uhm mine counts the rewatches. But I tried viewing other users stats and the call to the api returns all the animes with `my_rewatching` set to 0. (I did this by just by changing the username at the top of the `list` in `api.py`)

**Comment by ryukinix** [comment]
merged into `dev` #43
